.github/workflows/snyk.yml

```diff
@@ -0,0 +1,25 @@
+name: "Snyk"
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    # The branches below must be a subset of the branches above
+    branches: [ main ]
+  schedule:
+    - cron: '44 18 * * 3'
+
+jobs:
+  security:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+      - uses: snyk/actions/setup@0.3.0
+      - uses: actions/setup-go@v3.0.0
+        with:
+          go-version: 1.17.x
+      - name: Snyk monitor
```

Comment:
snyk "monitor" is a different command than "snyk test", no?

Reply:
The docs are shitty, but yes. the name of the task is wrong.